scoreboard/utils.py

```diff
@@ -22,6 +22,9 @@
 import pytz
 import urlparse
 
+from random import SystemRandom
+random = SystemRandom()
```

Comment:
Since `SystemRandom()` acts as a drop-in replacement for `random,` it seemed reasonable to name it random. However I don't know if this is considered good style